lua/entities/gmod_tardis/modules/sv_handbrake.lua

```diff
@@ -28,9 +28,9 @@ ENT:AddHook("ShouldFailDemat", "handbrake", function(self, force)
 end)
 
 ENT:AddHook("HandbrakeToggled", "vortex", function(self, on)
-    if on and self:GetData("teleport") or self:GetData("vortex") then
+    if on and (self:GetData("teleport") or self:GetData("vortex")) then
```

Comment:
Would be easier to read if we return early `if not on` rather than this

Reply:
makes sense
8f56cf4a5a6392263641bc9f306fb854932b7440